src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -32,10 +32,10 @@ Basic requirements:
 Compilers we support (listed with minimum supported version):
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  * Clang 3.9.1
+  * Clang 4.0.0
```

Comment:
Do we want to maintain this sort of thing?

Reply:
I definitely fall under the rip a lot of this documentation out...... so no